setup.py

```diff
@@ -1,6 +1,19 @@
 import os
 import sys
 
+if sys.version_info[:2] < (3, 5):
```

Comment:
https://python3statement.org/practicalities/ recommends raising a helpful import time exception should users with unsupported python versions successfully install the wrong package for them.

However, I suspect this code wont be used given that under the following circumstances quandl will fail to install al together given that some dependencies require python >= 3.5
- Outdated pip
- Python 2 or < 3.5